src/domain/Exercise.java

```diff
@@ -1,7 +1,23 @@
-package domain;
-
-public class Exercise {
-    public static float Calculate(int R){
-        return (float) (2*Math.PI*Math.sqrt(R));
-    }
-}
+package domain;
+/**This class calculate Ackerman functions
+ * @see TestResult
+ * @author Ivan
+ */
+public class Exercise {
+    /**
+     * 
+     * @param n first positive number
+     * @param m second positive number
+     * @return integer value of Calculate
+     */
+    public static int Calculate(int n, int m){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.